handler.ts

```diff
@@ -157,56 +90,49 @@ export const makeGeocoderRequests = async (
   delete peliasQSP.layers
 
   // Run both requests in parallel
-  let [primaryResponse, customResponse]: [
-    FeatureCollection,
-    FeatureCollection
-  ] = await Promise.all([
-    cachedGeocoderRequest(
-      getPrimaryGeocoder(),
-      apiMethod,
-      convertQSPToGeocoderArgs(event.queryStringParameters),
-      // @ts-expect-error Redis Typescript types are not friendly
-      redis
-    ),
-    // Custom request is either through geocoder package or "old" pelias method
-    getTransitGeocoder()
-      ? cachedGeocoderRequest(
-          getTransitGeocoder(),
-          'autocomplete',
-          convertQSPToGeocoderArgs(event.queryStringParameters),
-          null
+  let responses: FeatureCollection[] = await Promise.all(
+    geocoders.map((geocoder) =>
+      cachedGeocoderRequest(getGeocoder(geocoder), apiMethod, {
+        ...convertQSPToGeocoderArgs(event.queryStringParameters),
+        items: pois
+      })
+    )
+  )
+
+  responses = await Promise.all(
```

Comment:
Make `responses` a const and name this one something else to describe what's happening here.